<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayoutSettings</class>
 <widget class="QWidget" name="LayoutSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>589</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="scriptLbl">
     <property name="text">
      <string>Script:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="scriptEditor"/>
   </item>
   <item>
    <widget class="QGroupBox" name="colorsBox">
     <property name="title">
      <string>Colors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="greyBtn">
        <property name="text">
         <string>Grey color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="greyVal"/>
      </item>
      <item>
       <widget class="QRadioButton" name="tricolorBtn">
        <property name="text">
         <string>Tricolor</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="colorsLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="redLbl">
          <property name="text">
           <string>Red</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="redVal"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="greenLbl">
          <property name="text">
           <string>Green</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="greenVal"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="blueLbl">
          <property name="text">
           <string>Blue</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="blueVal"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="systemBtn">
       <property name="text">
        <string>Coordinats</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="systemLbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="pointerGroup">
     <property name="title">
      <string>Pointer Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="XLbl">
        <property name="text">
         <string>X axis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="xrelate">
        <property name="text">
         <string>Relatively</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="xabsolute">
        <property name="text">
         <string>Absolute</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="YLbl">
        <property name="text">
         <string>Y axis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="yrelate">
        <property name="text">
         <string>Relatively</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="yabsolute">
        <property name="text">
         <string>Absolute</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QPushButton" name="renderBtn">
       <property name="text">
        <string>Render</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
